frontend/src/sass/components/_footer.scss

```diff
@@ -1,4 +1,4 @@
-.footer {
+.global-footer {
 	background: $lighter-black-colour;
 	
 	&__list {
```

Comment:
Yeah, I'm more used to 2 spaces school. If you used 4 spaces throughout the project, then no worries!